functions/migrations/handler.ts

```diff
@@ -4,8 +4,8 @@ import middy from "@middy/core";
 import { awsLambdaResponse } from "../../shared/aws";
 import { winstonLogger } from "../../shared/logger";
 import { ConnectionManager } from "../../shared/utils/connection-manager";
-import { handleError } from "../../shared/middleware/error-handler";
-import { logIncomingEvent } from "../../shared/middleware/log-incomming-event";
+import { inputOutputLoggerConfigured } from "../../shared/middleware/input-output-logger-configured";
+import { customHttpErrorHandler } from "../../shared/middleware/custom-http-error-handler";
 
 export const handle = middy(async function handle(__: any, _: Context): Promise<any> {
```

Comment:
`_: APIGatewayProxyEvent, _:Context`

Reply:
It is not triggered by API gateway so i don't think this is the correct type